logging error?

```bash
/home/ironman/unomp/unified/libs/poolWorker.js:225
            logger[severity](logSystem, logComponent, logSubCat, text);
                            ^
TypeError: Object #<Logger> has no method 'special'
    at pool.on.on.on.on.process.send.type (/home/ironman/unomp/unified/libs/poolWorker.js:225:29)
    at pool.emit (events.js:98:17)
    at emitSpecialLog (/home/ironman/unomp/unified/node_modules/merged-pool/lib/pool.js:28:49)
    at OutputPoolInfo (/home/ironman/unomp/unified/node_modules/merged-pool/lib/pool.js:133:9)
    at /home/ironman/unomp/unified/node_modules/merged-pool/lib/pool.js:51:29
    at StratumServer.StartStratumServer._this.stratumServer.on.on.on.client.on.on.extraNonce2Size (/home/ironman/unomp/unified/node_modules/merged-pool/lib/pool.js:573:13)
    at StratumServer.emit (events.js:92:17)
    at Server.<anonymous> (/home/ironman/unomp/unified/node_modules/merged-pool/lib/stratum.js:439:27)
    at Server.g (events.js:180:16)
    at Server.emit (events.js:117:20)
[2015-02-22 14:33:30.398] [ERROR] [default] - Master PoolSpawner Fork 0 died, spawning replacement worker...
```
Website still runs. Kills stratum afaik.

--- a/libs/poolWorker.js
+++ b/libs/poolWorker.js
@@ -222,7 +222,7 @@ module.exports = function(logger){
             logger.debug(logSystem, logComponent, logSubCat, 'Difficulty update to diff ' + diff + ' workerName=' + JSON.stringify(workerName));
             handlers.diff(workerName, diff);
         }).on('log', function(severity, text) {
-            logger[severity](logSystem, logComponent, logSubCat, text);
+            logger.debug(logSystem, logComponent, logSubCat, text);
         }).on('banIP', function(ip, worker){
             process.send({type: 'banIP', ip: ip});
         }).on('started', function(){

